test/framed/std/io_test.clj

```diff
@@ -20,3 +20,14 @@
                 ostream (io/output-stream f2)]
       (std.io/stream-copy istream ostream))
     (is (= contents (slurp f2)))))
+
+(deftest test-copy
+  (let [src (std.io/spit (std.io/tempfile) "Hello,World")]
+    (testing "when dest does not exist"
+      (let [dest (std.io/copy src (io/file (std.io/temp-path)))]
+        (is (true? (.exists src)) "It does not modify src")
```

Comment:
This seems like it can only test for "it does not *delete* src," but this isn't testing that it doesn't *modify* src.